packages/react-ai/src/components/AIConversation/views/default/Form.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import {
+  Button,
+  DropZone,
+  TextAreaField,
+  View,
+  VisuallyHidden,
```

Comment:
Same questions as with other amplify components usage

Reply:
These are the default amplify components which use our primitives